hammers/jackhammer

```diff
@@ -501,8 +517,20 @@ if __name__ == '__main__':
     dump_parser = subparsers.add_parser('dump', help='Dumps all docker logs')
     dump_parser.add_argument('slots', nargs='*', type=int,
                              help='Specifies the slots to dump rogue states')
+    dump_parser.add_argument('--dump-rogue', action='store_true',
+                             help="If true, will attempt to connect to pysmurf smurf and dump the rogue tree")
     dump_parser.set_defaults(func=dump_func)
 
+    ########### Jackhammer dump parser ###########
+    dump_parser = subparsers.add_parser('deactivate', help='Deactivates slots')
+    dump_parser.add_argument('slots', nargs='*', type=int,
+                             help='Specifies the slots to deactivate')
+    dump_parser.set_defaults(func=deactivate_func)
+
+    dump_parser = subparsers.add_parser('activate', help='Activates slots')
+    dump_parser.add_argument('slots', nargs='*', type=int,
+                             help='Specifies the slots to activate')
+    dump_parser.set_defaults(func=activate_func)
```

Comment:
A few questions...
1) Why is the activate/deactivate slots part of the "dump_parser" instead of "hammer_parser" or its own section in here?
2) What is the default action if you don't pass it a slot or list of slots? 

I think if my first question doesn't matter and the 2nd one you fix by making it fail in a reasonable way then I'm happy to accept and merge.

Reply:
1. Good catch. Turns out the variable name of the parser doesn't actually matter since I'm creating a new one, but I will rename to match current style
2. I'll take Daniel's suggestion and change nargs to '+'